server.js

```diff
@@ -45,6 +45,13 @@ app.prepare().then(() => {
             : pathname)
       })
       res.end()
+    } else if (req.headers.host === 'err.dvc.org') {
+      // err.dvc.org/{hdr} -> dvc.org/doc/user-guide/troubleshooting#{hdr},
+      res.writeHead(301, {
+        'Cache-Control': 'no-cache',
+        Location: `https://dvc.org/doc/user-guide/troubleshooting#${pathname}`
```

Comment:
we will have to keep one->one mapping between title in the Markdown and the error code, right? The "code" will look like "too-many-files" at least

Reply:
Yes, the anchor is `too-many-open-files-error` actually. I think we **could** force it to `many-errors` with HTML in the Markdown though, what do you think? This way we can avoid having to set a redirect for each error path->header.

Reply:
👍 if we can force it, otherwise change title to be way smaller

Reply:
Hmmm. OK I forced it in ad1b786f and it works with the desired URL hash, see https://dvc-landing-2473-amnjganfpaic8.herokuapp.com/doc/user-guide/troubleshooting#many-files

> p.s. @pared notice I moved the file to its new correct location (public/static/doc/...)

BUT the special 🔗  icon and link stopped working. I guess those are automatically rendered by `react-markdown`.

Is if OK without the clickable header links and that anchor symbol?

Reply:
(let me know what name you decide to keep err, error, or something else, I'll setup the DNS record for it)

Reply:
OK can I just get a confirmation that this ^ is OK with others before merging the PR? Thanks

Reply:
could you summarize the `^`? :)

Reply:
Sure. So the redirect is error.dvc.org/{title} -> dvc.org/doc/user-guide/troubleshooting#{title}. The question here is about `title`.

In the website, it would naturally by a slugified version of the md header text, so in this case **Too many open files error** -> `too-many-open-files-error`. We could simply use that in the redirect BUT a) it's kind of long and b) what if we change the title later? We would need to create a new redirect each time.

We also previously agreed to use some textual codes for this, such as `many-files` in this case. But in our engine, to enforce that `id` prop of the actual `<h3>` element, we need to hardcode HTML code instead of markdown (current version). This works but the rendering is not as nice or functional as with MD – That's what I was explaining in https://github.com/iterative/dvc.org/pull/875#discussion_r365380109 above.

Reply:
> But in our engine, to enforce that id prop of the actual <h3> element, we need to hardcode HTML code

Wait... I just thought about another solution! I guess we could add hidden HTMl anchors before regular markdown headers and have the best of both worlds! (Except that clicking on the 🔗 link will produce a different hash than the tiny version from the `error...` redirect)

Reply:
@jorgeorpinel we can create a ticket to improve the engine. It's totally fine. For now hidden HTML and/or hardcoding it totally fine.

Reply:
OK I hardcoded the whole thing by just copying the code that `react-markdown` would render, and changing the `id` and `href` HTML attributes. Please check, and I guess merge this if OK?

Reply:
p.s. now deployed: https://dvc-landing-2473-amnjganfpaic8.herokuapp.com/doc/user-guide/troubleshooting#many-files looks and works perfectly (it's just hardcoded so future updates of `react-markdown` could possibly cause problems with styles or something).